repository-ydb-v2/src/main/java/tech/ydb/yoj/repository/ydb/table/YdbTable.java

```diff
@@ -54,18 +57,36 @@
 import static tech.ydb.yoj.repository.db.EntityExpressions.defaultOrder;
 
 public class YdbTable<T extends Entity<T>> implements Table<T> {
+    private static final AtomicBoolean useOldStatementFactory = new AtomicBoolean(Boolean.getBoolean("yoj.use.type.statement.factory"));
+
+    @Deprecated(forRemoval = true)
+    public static void setUseOldStatementFactory(boolean value) {
```

Comment:
A comment for @lavrukov: 

If you don't use custom table names, the `YqlStatementFactory.Descriptor` statement factory *should* produce the same `YqlStatements` as previous versions of YOJ. 

But out of **abundance of caution**, we added a way to construct `YqlStatements`  only using entity class, not (entity class [+ optional table name]). To use this legacy `YqlStatementFactory` implementation, set a Java property `yoj.use.type.statement.factory` to `true` or explicitly call `YdbTable.setUseOldStatementFactory(true)` before using YOJ.